ARM-wvd-templates/nestedtemplates/managedDisks-customimagevm.json

```diff
@@ -7,7 +7,7 @@
             "metadata": {
                 "description": "The base URI where artifacts required by this template are located."
             },
-            "defaultValue": "https://wvdportalstorageblob.blob.core.windows.net/galleryartifacts/Configuration.zip"
+            "defaultValue": "https://raw.githubusercontent.com/Azure/RDS-Templates/master/ARM-wvd-templates/DSC/Configuration.zip"

```

Comment:
Could you explain what's the reason for switching this URL?

Reply:
Ideally both locations are in sync, but they are actually 2 different copies.  Therefore it should point to github location here.